README.md

```diff
@@ -7,6 +7,16 @@ Contao Swiper
 
 Contao 4+ bundle to integrate [Swiper.js](http://idangero.us/swiper/).
 
+This extension replaces the core slider with a Swiper implementation.
+
 ## Usage
 
-__jQuery__ needs to be enabled in your page layout. This extension replaces the core slider with a Swiper implementation.
+__jQuery__ does **not** have to be enabled in your page layout.
+
+| Feature | Description |
+|----|----|
+| *multiple sliders* | adding multiple sliders with different settings on a page |
+| *different slider-effects* | slide, fade, cube, coverflow, flip |
+| *different pagination-types* | bullets, fraction, progressbar |
+| *breakpoints* | change some settings on different viewports |
+| *custom wrapper-class* | Helpful when adding a *module* (e.g. newslist-module) inside the slider. You can define the `mod-newslist`-class as wrapper for the slides, so that it is possible to slide through the single news-entries. |
```

Comment:
Should me `mod_newslist` not `mod-newslist` ;)

Reply:
corrected the class-name.